workspace/__lib__/xod/core/cast-boolean-to-pulse/patch.xodp

```diff
@@ -1,6 +1,23 @@
 {
   "description": "Emits a pulse on a rising edge, i.e. when false changes to true",
   "nodes": [
+    {
+      "id": "H19x_iG9G",
+      "position": {
+        "x": 170,
+        "y": 102
+      },
+      "type": "xod/patch-nodes/utility"
+    },
+    {
+      "description": "Use xod/core/cast or its specialization.",
```

Comment:
I would say we don’t want to even encourage the direct use of `cast*` nodes. So the better message here and in other places is:

> Use `xod/core/pulse-on-true` instead

Reply:
👌 
Updated deprecated reasons for old-fashioned `cast` nodes:
- `boolean-to-number` -> `if-else`
- `boolean-to-string` -> `if-else`
- `boolean-to-pulse` -> `pulse-on-true`
- `number-to-boolean` -> `equal`
- `number-to-string` -> `format-number`